controllers/organizations/organization.js

```diff
@@ -70,9 +71,6 @@ module.exports = {
       .select('uid')
       .catch(err => next(err));
 
-    if (!publicOrgs)
```

Comment:
Do empty queries return empty array? Can they be checked with not operator or only by checking length of array?